src/hooks/users/useConnectWorldUsers.ts

```diff
@@ -0,0 +1,29 @@
+import { useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSovereignVenueId } from "hooks/useSovereignVenueId";
+import { useVenueId } from "hooks/useVenueId";
+
+export const useConnectWorldUsers = () => {
```

Comment:
[note] This hook was split out of `src/hooks/users/index.ts` into it's own file here.